### ISSUE TITLE
 Error during finetuning: MisconfigurationException: The provided lr scheduler LambdaLR doesn't follow PyTorch's LRScheduler API

### ISSUE THREAD

**Issue by Edwarddev0723** [bcmi/MureObjectStitch-Image-Composition#1]
---

#### Description:
I encountered an error while running the finetuning script. Below is the detailed error message and environment setup.

---

#### Error Log:
```
Running on GPUs 1
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.54 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 64, 64) = 16384 dimensions.
making attention of type 'vanilla' with 512 in_channels
FrozenCLIPImageEmbedder has 366.16 M params.
Load  /content/MureObjectStitch-Image-Composition/checkpoints/ObjectStitch.pth
Monitoring val/loss_simple_ema as checkpoint metric.
Merged modelckpt-cfg: 
{'target': 'pytorch_lightning.callbacks.ModelCheckpoint', 'params': {'dirpath': '/content/MureObjectStitch-Image-Composition/experiments/objectstitch/2025-01-09T07-48-32_car_multifg/checkpoints', 'filename': '{epoch:06}', 'verbose': True, 'save_last': False, 'save_weights_only': True, 'monitor': 'val/loss_simple_ema', 'save_top_k': 5}}
...
[rank0]: MisconfigurationException: The provided lr scheduler LambdaLR doesn't follow PyTorch's LRScheduler API. You should override the LightningModule.lr_scheduler_step hook with your own logic if you are using a custom LR scheduler.
```

---

#### Steps to Reproduce:
1. Clone the repository and follow the setup instructions.
2. Run the training script using the following command:
   ```bash
   sh finetune.sh
   ```
3. The error occurs during the initialization of the training process.

---

#### Observations:
- The error suggests that the `LambdaLR` scheduler doesn't adhere to the PyTorch's LRScheduler API.
- This issue might be related to the recent updates in PyTorch Lightning or PyTorch itself.
- Warning logs also indicate other library compatibility issues, e.g., `Albumentations`, `torch.load` deprecation warnings.

---

#### Environment Configuration:
- **PyTorch version:** `2.5.1+cu121`
- **PyTorch Lightning version:** `1.8.1`
- **Python version:** `3.10`
- **CUDA version:** `12.1`
- **GPU:** NVIDIA A100 (4 GPUs)
---

**Comment by csdahunzi** [comment]
I think it might be due to the PyTorch version being too high. It is recommended to keep it consistent with the version specified in the README.
